guide/interactions/select-menus.md

```diff
@@ -187,15 +187,15 @@ client.on('interactionCreate', async interaction => {
 A select menu is not bound to only one selection; you can specify a minimum and maximum amount of options that must be selected. You can use `MessageSelectMenu#setMinValues()` and `MessageSelectMenu#setMaxValues()` to determine these values.
 
 ```js {1,7-31,33}
-const { MessageActionRow, MessageSelectMenu } = require('discord.js');
+const { ActionRowBuilder, SelectMenuBuilder } = require('discord.js');
 
 client.on('interactionCreate', async interaction => {
-	if (!interaction.isCommand()) return;
+	if (!interaction.isChatInputCommand()) return;
```

Comment:
I find it odd that the text at the top states to use `CommandInteraction#reply()` yet we are only allowing chat input interactions through. This will work with context menus too.